documentation/modules/ostack-app-cred-auth.adoc

```diff
@@ -92,4 +92,4 @@ stringData:
   url: <OS_AUTH_URL_from_openstack_rc_file>
 EOF
 ----
-. Continue migrating your virtual machine according to the procedure in xref:new-migrating-virtual-machines-cli_ostack[Migrating virtual machines], starting with step 2, "Create a `Provider` manifest for the source provider."
+// . Continue migrating your virtual machine according to the procedure in xref:new-migrating-virtual-machines-cli_ostack[Migrating virtual machines], starting with step 2, "Create a `Provider` manifest for the source provider."
```

Comment:
@anarnold97 I commented out this sentence because the xref didn't work as it used to. I tried a number of fixes, but none worked.